app/src/main/java/secapstone/helper/Listing.java

```diff
@@ -14,6 +14,8 @@
     private String description = "";
     private ProductCategory category;
 
+    public float productID;
```

Comment:
add private price(float) and shipping method (boolean) fields and getter methods